packages/opencensus-exporter-stackdriver/README.md

```diff
@@ -55,6 +55,42 @@ or
 tracing.registerExporter(exporter).start();
 ```
 
+## Monitoring Exporter
+
+To use Stackdriver as your exporter, make sure you have enabled [Stackdriver Monitoring](https://cloud.google.com/monitoring/docs/quickstart) on Google Cloud Platform. Enable your [Application Default Credentials](https://cloud.google.com/docs/authentication/getting-started) for authentication with:
+
+```bash
+export GOOGLE_APPLICATION_CREDENTIALS=path/to/your/credential.json
+```
+
+Instance the exporter on your application passing your Project ID and register your exporter on Stats. For javascript:
+
+```javascript
+var opencensus = require('@opencensus/core');
```

Comment:
I think we should use ES6 wherever possible.

Reply:
Sure!